app/jobs/background_render/cas_readiness_job.rb

```diff
@@ -0,0 +1,46 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+class BackgroundRender::CasReadinessJob < BackgroundRenderJob
+  queue_as ENV.fetch('DJ_SHORT_QUEUE_NAME', :short_running)
+
+  def render_html(client_id:, user_id:, token:)
+    current_user = User.find(user_id)
+    @client = GrdaWarehouse::Hud::Client.destination_from_searchable_to(current_user).
+      preload(
+        :source_exits,
+        source_enrollments: [
+          :exit,
+          :income_benefits,
+        ],
+      ).
+      find(client_id.to_i)
+
+    ActionController::Renderer::RACK_KEY_TRANSLATION['warden'] ||= 'warden'
+    warden_proxy = Warden::Proxy.new({}, Warden::Manager.new({})).tap do |i|
+      i.set_user(current_user, scope: :user, store: false, run_callbacks: false)
+    end
```

Comment:
looks like we have 17 other instances where we use 'ActionController::Renderer'. Maybe we could DRY up this pattern in a wrapper class?
```ruby
# RailsHeadlessController.new(ReportController, user: user).render(partial: 'reports')
 class RailsHeadlessController
    def initialize(controller_class, user: nil)
      @controller_class = controller_class
      @user = user
    end

    def render(...)
      renderer.render(...)
    end

    def renderer()
      @controller_class.renderer.new('warden' => warden_proxy)
    end

    def  warden_proxy
      ActionController::Renderer::RACK_KEY_TRANSLATION['warden'] ||= 'warden'
      Warden::Proxy.new({}, Warden::Manager.new({})).tap do |i|
        i.set_user(@user, scope: :user, store: false, run_callbacks: false)
      end
    end
  end
```

Reply:
Creating this issue for future cleanup:
https://github.com/open-path/Green-River/issues/6430